### ISSUE TITLE
[Add Content]: Mini ML Project to predict the house price

### ISSUE THREAD

**Issue by seeratfatima19** [animator/learn-python#597]
I propose to make ML project using TensorFlow or Scikit-Learn (as per your preference) to predict house prices using the house price dataset.

This project will be aimed to clear the concepts of classification and regression in machine learning.

Kindly assign me this issue under the label GSSoC'24

thanks and regards
